tests/unit/test_adapter.py

```diff
@@ -320,10 +322,15 @@ def test_parse_relation(self):
         input_cols = [Row(keys=["col_name", "data_type"], values=r) for r in plain_rows]
 
         config = self._get_target_http(self.project_cfg)
-        rows = SparkAdapter(config).parse_describe_extended(relation, input_cols)
-        self.assertEqual(len(rows), 4)
+        adapter = SparkAdapter(config)
```

Comment:
@Fokko : Could you rewrite this code to have a similar flow as before, i.e. (re)move the three build-up lines and have *one* call to get the to-be-tested object